recipes/liboping/all/patches/1.10.0-suppress_truncate.patch

```diff
@@ -0,0 +1,18 @@
+--- a/src/liboping.c	2022-01-04 12:15:14.312483115 -0700
++++ b/src/liboping.c	2022-01-04 12:16:40.031084749 -0700
+@@ -203,8 +203,15 @@
+ static void ping_set_error (pingobj_t *obj, const char *function,
+ 	       	const char *message)
+ {
++#if __GNUC__ >= 7
++#pragma GCC diagnostic push
++#pragma GCC diagnostic ignored "-Wformat-truncation"
```

Comment:
that's fine, but I guess patch could be avoided by passing `-Wno-format-truncation` from the outside

Reply:
Understood. I sent the patch to the other repo - there are possible other approaches - I will take whatever the original liboping repo accepts and update it to the post 1.10.0 release once they do w/o the patch.